demo/workflow/workflow.md

```diff
@@ -0,0 +1,722 @@
+# Expanding your view of the software supply chain
+
+### **Bruce Wayne** : You're just SBOMs.
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233477844-8f4f1743-654d-4c53-a6ff-23e0bcf3a6eb.jpg">
+
+### **Henri Ducard (Ra's al Ghul)** : No, no, no. An SBOM is just a document lost in the scramble for its own importance. It can be misplaced or underutilized. But if you make the SBOM more than just a document, you devote the SBOM to be utilized properly to secure your software supply chain, and if the attackers cannot get to you via supply chain attacks, then the SBOM becomes something else entirely.
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233478647-a62fccb0-1e87-4542-b089-2091155c553c.png">
+
+### **Bruce Wayne** : Which is?
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233477213-f6d99be5-f669-4344-8c1e-123f063db3de.png">
+
+### **Henri Ducard (Ra's al Ghul)** : Legend, Mr. Wayne!
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233479006-9fa20a7e-1a13-41d1-9f97-2345fcd85c69.png">
+
+Ra’s al Ghul understands that an SBOM alone can be forgotten but if you combine
+it with GUAC, you can start to get a greater understanding of your own software
+supply chain environment.
+
+In this demo, we will go through the process of ingesting an SBOM and letting
+GUAC expand our horizons on what we know about our environment autonomously!
+
+## Requirements
+
+- go
+- Kubernetes 1.19+ ([k3s](https://github.com/k3s-io/k3s/),
+  [minikube](https://github.com/kubernetes/minikube) or
+  [colima](https://github.com/abiosoft/colima) (or another k8s service of your
+  choice))
+- [Helm v3.9.4+](https://helm.sh/)
+
+## Clone GUAC
+
+If you haven't already, clone GUAC to a local directory:
+
+```bash
+git clone https://github.com/guacsec/guac.git
+```
+
+Also, clone GUAC data, this is used as test data for this demo.
+
+```bash
+git clone https://github.com/guacsec/guac-data.git
+```
+
+The rest of the demo will assume you are in the GUAC directory
+
+```bash
+cd guac
+```
+
+## Building the GUAC binaries
+
+Build the GUAC binaries using the `make` command.
+
+```bash
+make
+```
+
+## Installing GUAC via Helm Chart
+
+Please refer to the [INSERT GUAC Helm Install Guide] to have a running instance.
```

Comment:
can note docker compose alternativw here except its a bit more difficult to observe individiaul components

Reply:
added